awscli/examples/internetmonitor/create-monitor.rst

```diff
@@ -0,0 +1,16 @@
+**To create a new internet monitor**
+
+This example uses the ``create-monitor`` command to create a new internet monitor. ::
```

Comment:
```suggestion
The following ``create-monitor`` example creates a new internet monitor. ::
```

All your examples need their intro lines updated to follow the format of "The following ``command-name`` example..."
https://aws.github.io/aws-cli/docs_styleguide.html#description